### PR TITLE
Adds Non-Eutactic Blades and Variations

### PR DESCRIPTION
**Pull request by Michiyamenotehifunana**
:cl: Toriate
add: Added the non-eutactic blade, fancy recolorable esword
add: Added the hyper-eutactic blade, supermassive recolorable desword
soundadd: added NEB sounds
imageadd: added NEB icons
/:cl:

dreamseeker_2017-10-24_18-37-20

Non-Eutactic Blades (NEBs) are essentially, eswords except cool and fancy. It also rips off a different sci-fi universe. Their stats are different from regular eswords. They are also not implemented in any way whatsoever. The toy variants however, are craftable in-game with plastic, cable coils, and light tubes.

dreamseeker_2017-10-24_18-31-21

dreamseeker_2017-10-29_23-14-07

Also included are Hyper-Eutactic Blades, a two handed variant similiar to the dual esword.

dreamseeker_2017-12-01_20-25-49

The actual stats are excessively overpowered and untested.

dreamseeker_2017-12-02_00-01-06

As none of the real variants are actually obtainable normally, they would have to be bussed in for testing in prod.

DISCLAIMER: None of the things shown in the screenshots aside from the swords are actually added in this PR. So no cool glowy knight armors are included.

### CODE REVIEW

File: haven/code/game/objects/items/melee/eutactic_blades.dm [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+
+/obj/item/twohanded/hypereutactic/process()
+	if(wielded)
+		open_flame()
+	else
+		STOP_PROCESSING(SSobj, src)
+
+/obj/item/twohanded/hypereutactic/IsReflect()
+	if(wielded)
+		return 1
```
Comment (Kevak) [inline haven/code/game/objects/items/melee/eutactic_blades.dm]: Remember the Boolians. This should be TRUE if it isn't intended to be a var.

Comment (Kevak) [reply haven/code/game/objects/items/melee/eutactic_blades.dm]: So yah. Just. Go through and find all the boolians.

**Review by Kevak — CHANGES_REQUESTED** [review]

**Comment by Michiyamenotehifunana** [comment]
@Kevak should be fixed now

**Review by Kevak — APPROVED** [review]

**Comment by Kevak** [comment]
Looks good but I'm gonna wait on sorting out the conflict that's causing problems with Travis. P sure we merged something derpy.

**Comment by Amelia0010** [comment]
Update your branch with the master to fix the conflicts

**Comment by Partheo** [comment]
### Just finished testing this. Here are my thoughts and comments:

I love the amount of detail you've put into this PR. Small things like the impale on clumsiness, or the plastic craftable swords, and the hulk messages make this feel like a complete feature.

The sounds are loud, but so are the e-sword ones so I have no problem with that.

The hyper sword is hilarious. Way too much damage, so I'm glad this is staying as an admin-only item. We can try to find a good use for it eventually. Maybe some crazy lavaland loot? The sword definitely feels like a "final boss" drop item.

A recent PR changed how disability procs worked, so I updated the clumsiness check for you.

I feel like the message asking "Are you sure you want to change [the sword's] color?" is unecessary. You can just click cancel on the color selection dialogue. 

The hyper sword sprite was not centered, so I moved it cause it bugged me. Was that on purpose?

The sword attack animation only displays the hilt (off mode) icon when you hit something with it, unlike the e-sword. See this gif for more info:


#### Overall, pretty happy with this PR. Feels like a quality feature even though it's not used in game.

**Pull request merged**